vm/vm.go

```diff
@@ -38,6 +38,7 @@ type VM struct {
 	globals     []object.Object
 	frames      []*Frame
 	framesIndex int
+	top         int // maximum stack tops used
```

Comment:
Could we make this name a bit more descriptive - something like `maxFramesUsed` or `framesAllocated`? Thanks!

Reply:
I've changed the naming of the fields and rebase the commit